### PR TITLE
feat: exemple init

### PR DESCRIPTION
**Pull request by Simperfit**

### CODE REVIEW

File: exemples/admin/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <script src="node_modules/reflect-metadata/Reflect.js"></script>
+    <script src="node_modules/systemjs/dist/system.src.js"></script>
+    <script>
+        System.import('app').catch(function (err) {console.error(err);});
+    </script>
+</head>
+<body>
+<app>Loading...</app>
+</body>
+</html>
```
Comment (Gregcop1) [inline exemples/admin/index.html]: blank line at the EOF

File: exemples/admin/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "dependencies": {
+    "@angular": "^2.0.0-rc.1",
+    "hm-admin": "0.1.0",
+    "es6-shim": "^0.35.0",
+    "http-server": "^0.8.5",
+    "reflect-metadata": "0.1.3",
+    "systemjs": "0.19.26"
+  }
+}
```
Comment (Gregcop1) [inline exemples/admin/package.json]: blank line at the EOF

Comment (Gregcop1) [inline exemples/admin/package.json]: why setting a different package.json ?

File: exemples/admin/src/app.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    template: `<hm-admin-app></hm-admin-app>
+               `
+})
+export class App {
+
+    message = "";
+
+}
+
+bootstrap(App);
```
Comment (Gregcop1) [inline exemples/admin/src/app.ts]: blank line at the EOF

File: exemples/admin/package.json (hunk 2 above)
Comment (Simperfit) [inline exemples/admin/package.json]: don't we need one for the exemple app ?

Comment (Gregcop1) [inline exemples/admin/package.json]: The exemple is à part of this project. It allows dev to test visually the component. So the package.json is the same. But you can save all dependencies of exemple in devDependencies IMO

File: exemples/admin/index.html (hunk 1 above)
Comment (Simperfit) [inline exemples/admin/index.html]: I've added one

File: exemples/admin/src/app.ts (hunk 3 above)
Comment (Simperfit) [inline exemples/admin/src/app.ts]: i've added one

File: exemples/admin/package.json (hunk 2 above)
Comment (Simperfit) [inline exemples/admin/package.json]: So we can make something like a demo admin for a blog or a bookstore ?

**Comment by Simperfit** [comment]
ping @Gregcop1 RFR

**Pull request merged**